### DISCUSSION TITLE
Fedora RPM client unreachable

### DISCUSSION THREAD

**Discussion by rpgdev** [rustdesk/rustdesk#8227] [General]
### Bug Description

Running rustdesk-server via docker on a Debian server and using a Fedora flatpak and Android client I can access my Fedora desktop with 100% reliability from my Android client but I can't access my Fedora desktop reliably when the native RPM client is used instead. Maybe I can do so 1% of the times I tried because sometimes I can access it after stopping and starting the service but once I disconnect I can't access it again until I restart the service again (sometimes it does, sometimes it doesn't). It takes way longer to connect when it does too. Weirdly, the Android client DOES show the Fedora client is available with a green dot while the Android client looks with an orange dot from the Fedora RPM client. This has been happening since I started to use rustdesk back in december so I've been using the flatpak instead (again 100% reliably, same server and setup). It's a shame because I really want to use this headless on Linux and that isn't possible with the flatpak. 

### How to Reproduce

Install the RPM client rustdesk-1.2.3-2.x86_64.rpm on Fedora 39
Connect to it using Android client 1.2.3

### Expected Behavior

Connect to my Fedora client reliably same as it does with the flatpak

### Operating system(s) on local side and remote side

Android 14 -> Fedora 39

### RustDesk Version(s) on local side and remote side

1.2.3 -> 1.2.3-2

### Screenshots

N/A

**Comment by xlionjuan** [comment]
Try nightly
